db/src/Pos/DB/Txp/Logic/Local.hs

```diff
@@ -78,11 +81,13 @@ txProcessTransactionNoLock
        , MempoolExt m ~ ()
        )
     => Genesis.Config
+    -> TxValidationRules
     -> TxpConfiguration
     -> (TxId, TxAux)
     -> m (Either ToilVerFailure ())
-txProcessTransactionNoLock genesisConfig txpConfig = txProcessTransactionAbstract
+txProcessTransactionNoLock genesisConfig _txValRules txpConfig = txProcessTransactionAbstract
```

Comment:
Why do we pass a `TxValidationRules` if we never use it?

Reply:
Same question.

Reply:
I managed to remove the unused `TxValidationRules` types, I tried before and ran into issues but they're gone now :).